src/main/java/org/jabref/cli/ArgumentProcessor.java

```diff
@@ -289,6 +294,45 @@ public void processArguments() {
         if (!cli.isBlank() && !loaded.isEmpty()) {
             uiCommands.add(new UiCommand.OpenDatabases(loaded));
         }
+
+        if (cli.isCheckConsistency()) {
+            checkConsistency();
+        }
+    }
+
+    private void checkConsistency() {
+        String fileName = cli.getCheckConsistency();
+        String outputFormat = cli.getCheckConsistencyOutputFormat().toUpperCase();
```

Comment:
`cli.getCheckConsistencyOutputFormat()` could be `null`? - Please handle. No need to use `toUpperCase()`. I give you a hint below.

```suggestion
        Optional<String> outputFormat = Optional.ofNullable(cli.getCheckConsistencyOutputFormat());
```

Update: I saw that you put TXT as default. Can you check if it is a default value somewhow? Otherwise, just remove the default and use my proposal.

--> Without `--check-consistency-output-format`: Write to console